src/Illuminate/Database/Connectors/ConnectionFactory.php

```diff
@@ -96,6 +97,58 @@ protected function createReadPdo(array $config)
      */
     protected function createPdoResolver(array $config)
     {
+        if (array_key_exists('host', $config)) {
+            return $this->createPdoResolverWithHosts($config);
+        }
+
+        return $this->createPdoResolverWithoutHosts($config);
```

Comment:
As far as I could see, sqlite configurations don't have a `host` property, hence the two different methods.